gas-service/src/lib.rs

```diff
@@ -110,6 +109,7 @@ pub trait GasService: events::Events {
     #[endpoint(payNativeGasForExpressCall)]
     fn pay_native_gas_for_express_call(
```

Comment:
I know it could be done this way, but I kept the same endpoints as in the reference Solidity implementation to be consistent with that:
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/gas-service/AxelarGasService.sol